topics/metagenomics/tutorials/metagenomics-binning/turorial.md

```diff
@@ -0,0 +1,418 @@
+>    - {% icon param-collection %} *"Graphical Fragment Assembly"*: Output of **megahit contig2fastg**
+>
+> 3. Inspect the generated image for ERR2231571
+{: .hands_on}
+
+![Image shows the assembly graphs, with longer stretch on the top and many small contigs on the bottom](./images/ERR2231571_graph.jpg "Assembly graph for ERR2231571 sample")
+
+The graph is quite disconnected. On the top, we can see the longer stretches, that includes multiples contigs (each contig having a different color). On the bottom are the shortest stretches or single contigs.
+
+But it is really hard to read or extract any information from the graph. Let's inspect the information about the assembly graph
+
+> <hands-on-title>Visualization the *de novo* assembly graph</hands-on-title>
+>
+> 1. {% tool [Bandage Info](toolshed.g2.bx.psu.edu/repos/iuc/bandage/bandage_info/0.8.1+galaxy2) %} with parameters:
+>    - {% icon param-collection %} *"Graphical Fragment Assembly"*: Output of **megahit contig2fastg**
+>
+> 2. {% tool [Column join](toolshed.g2.bx.psu.edu/repos/iuc/collection_column_join/collection_column_join/0.0.3) %} with parameters:
+>    - {% icon param-collection %} *"Tabular files"*: Output of **Bandage Info**
+>
+> 3. Inspect the generated output
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How many nodes are in the graph for ERR2231568? And for ERR2231572? What does they correspond to?
+> 2. How many edges are in the graph for ERR2231568? And for ERR2231572? What is the impact of these numbers in relation to the number of nodes on the graph?
+> 3. How many connected components are there for ERR2231568? And for ERR2231572? What does they correspond to?
+> 4. What is the percentage of dead ends are there for ERR2231568? And for ERR2231572?
+> 5. What are the smallest and larges edge overlaps?
+> 6. What is the largest component? For which sample?
+> 7. What is the shortest node? What does they correspond to?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. There are 228,719 nodes for ERR2231568 and 122,526 for ERR2231572. They correspond to the number of contigs
+> > 2. There are 16,580 edges for ERR2231568 and 13,993 for ERR2231572. There are less edges than nodes in the graph. It means that many nodes/contigs are disconnected
+> > 3. There are 212,598 connected components, i.e. number of regions of the graph which are disconnected from each other, for ERR2231568 and 109,044 for ERR2231572
+> > 4. There are 94.0702% dead ends, i.e. the end of a node not connected to any other nodes, for ERR2231568 and 90.7032% for ERR2231572. It confirms the previous observation
+> > 5. The smallest and larges edge overlaps are 91bp, i.e. the k-mer length
+> > 6. The largest component is 340,003 bp for ERR2231567
+> > 7. The shortest node is 200 bp, i.e. the minimal size for a contig
+> >
+> {: .solution}
+>
+{: .question} -->
+
+<!--# De-replication
```

Comment:
It would be interesting to cover dereplication in this tutorial

Reply:
I would put a section about dereplication. It could be mentioned that MetaBAT2 is doing it, but there is also tool for that and then explaining how to do it using dRep

Reply:
A dereplication section has been added. Could you please review it and provide feedback? Also, if I remember correctly there was a previous issue with site construction. Is this now resolved?